packages/python-packages/doc-warden/warden/warden_common.py

```diff
@@ -5,7 +5,7 @@
 import fnmatch
 import re
 import xml.etree.ElementTree as ET
-import pathlib
+import pathlib2 as pathlib
```

Comment:
This is definitely easier. Given that we just say `pathlib.Path` I think it'll be better if we replace the actual reference. Aka. `pathlib2.Path` 

It's a pain, but I do think it'll be much more visible and there will be zero chance for confusion down the line.